mmv1/templates/terraform/examples/oracledatabase_autonomous_database_basic.tf.tmpl

```diff
@@ -0,0 +1,23 @@
+resource "google_oracle_database_autonomous_database" "{{$.PrimaryResourceId}}"{
+  display_name = "autonomousDatabase displayname"
+  autonomous_database_id = "my-adb-instance"
+  location = "us-east4"
+  project = "{{index $.Vars "project"}}"
+  database = "testADBdb"
```

Comment:
Can we omit the `optional` values here?

Reply:
removed displayName, we need to pass project as we want to override the CI's project with our GCP project.